src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs

```diff
@@ -5,26 +5,35 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace BlazorHosted_CSharp.Server.Controllers
+namespace BlazorHosted_CSharp.Server.Controller
 {
-    [Route("api/[controller]")]
-    public class SampleDataController : Controller
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
     {
-        private static string[] Summaries = new[]
+        private static readonly string[] Summaries = new[]
         {
             "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
         };
 
-        [HttpGet("[action]")]
-        public IEnumerable<WeatherForecast> WeatherForecasts()
+        private readonly ILogger<WeatherForecastController> _logger;
```

Comment:
Why inject the logger if it’s not used?

Reply:
Great Question @SteveSandersonMS. Some questions are best left unanswered.